### DISCUSSION TITLE
🔗 Compatibility of woocommerce notice widget with popup (Displaying WooCommerce notice in popup) (Merged with #25712)

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21623] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Elementor is now ready to release a great feature for WooCommerce. **Because Elementor now has both woocommerce notice widget and popup.** Therefore, the release of the ability to display Woocommerce notice as a pop-up in Elementor does not seem to be a difficult task (in terms of creating context).

### Describe the solution you'd like

Please make woocommerce notice compatible with popup. This would be a really great feature. This is a game changer for e-commerce websites.

I hope many users will vote for this request and it will be added to the Elementor roadmap.

**Comment by nicholaszein** [comment]
Hello there, @Joliamendi!

Thank you for your submission! 🙏 

### 🔗 🚀 There is a similar request submitted by someone here #25712 

For now, we are merging this request with the other one.

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

(Don't worry, we will count the votes here 😉)

Kind regards
